src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/AggregateCrossTargetProjectContextProvider.cs

```diff
@@ -202,7 +202,7 @@ private void AddConfiguredProjectState(ConfiguredProject configuredProject, ITar
         private async Task<AggregateCrossTargetProjectContext> CreateProjectContextAsyncCore()
         {           
             // Don't initialize until the project has been loaded into the IDE and available in Solution Explorer
-            await _asyncLoadDashboard.ProjectLoadedInHost.ConfigureAwait(false);
+            await _asyncLoadDashboard.ProjectLoadedInHostWithCancellation(_commonServices.Project).ConfigureAwait(false);
```

Comment:
Was trying to enforce correct coding pattern (use `UnloadCancellationToken`, as opposed to `CancellationToken.None`). I was going to name the method something like "SafeProjectLoadedInHost", but settled on keeping the word cancellation in the name since it is more descriptive